app/src/main/java/com/vci/vectorcamapp/intake/domain/use_cases/ValidateNumPeopleSleptInHouseUseCase.kt

```diff
@@ -0,0 +1,15 @@
+package com.vci.vectorcamapp.intake.domain.use_cases
+
+import com.vci.vectorcamapp.core.domain.util.Result
+import com.vci.vectorcamapp.intake.domain.util.FormValidationError
+import javax.inject.Inject
+
+class ValidateNumPeopleSleptInHouseUseCase @Inject constructor() {
+    operator fun invoke(count: Int): Result<Unit, FormValidationError> {
+        return if (count == -1) {
```

Comment:
Make sure this is consistent across all use cases. In some it says <= -1. In others it says == -1.